config/i18n.json

```diff
@@ -389,8 +389,8 @@
   "ItemMove": {
     "Consolidate": "Consolidated {{name}}",
     "Distributed": "Distributed {{name}}\n {{name}} is now equally divided between characters.",
-    "MovingItem_male": "Transfer to {{target}}",
-    "MovingItem_female": "Transfer to {{target}}",
+    "MovingItem_male": "Transferring to {{target}}",
+    "MovingItem_female": "Transferring to {{target}}",
```

Comment:
This seems unrelated. I had it as "Transfer" since it'll also show this message for a bit after the transfer is done - the progress notification is about the transfer itself.

Reply:
oh dim.json in master says "transferring" so i was just updating root string to match. undone.